platform-api/src/routers/platform.py

```
@@ -0,0 +1,118 @@
+    # Fast local checks
+    checks["kv"] = "ok" if settings.kv else "not_configured"
+    checks["d1"] = "ok" if settings.db else "not_configured"
+
+    status = "degraded" if any("error" in v or "not_configured" in v for v in checks.values()) else "ok"
+    return {"status": status, "checks": checks}
+
+@router.get("/analytics", response_model=AnalyticsResponse)
+async def analytics(
+    metric: str = Query("requests", description="Metric to fetch"),
+    period: str = Query("24h", description="Time period"),
+    settings: WorkerSettings = Depends(get_settings),
```

Ruff (0.15.11)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)